<launch>
  <arg name="layer_period" default="10" doc="period" />

  <arg name="map_frame_id" default="map" doc="map_frame_id" />
  <arg name="robot_frame_id" default="fixed_01" doc="frame_id and agent_id(mobile_01, mobile_02, fixed_01, fixed_02)" />
  <arg name="world_frame_id" default="world" doc="world_frame_id" />
  <arg name="pose_topic" default="/robot_odom" doc="tracks topic" />
  
	<node pkg="osr_pose" type="fixed_agent_pose_publisher.py" name="$(arg robot_frame_id)_agent_pose_publisher" output="screen">
	    <param name="period" value="$(arg layer_period)" />
	    <param name="map_frame_id" value="$(arg map_frame_id)" />
		<param name="robot_frame_id" value="$(arg robot_frame_id)" />
		<param name="world_frame_id" value="$(arg world_frame_id)" />

	    <param name="pose_topic" value="$(arg pose_topic)" />
	    <param name="pose_fliename" value="$(arg robot_frame_id)_to_map.txt" />
	</node>



</launch>
